### PR TITLE
Add accessibility statement

### PR DESCRIPTION
**Pull request by tombye**
Adds the accessibility statement for GOV.UK Notify.

Here's [a screenshot of what it looks like in a browser](https://user-images.githubusercontent.com/87140/94001090-edcce400-fd8f-11ea-839b-25abd90c911e.png)

### CODE REVIEW

File: app/templates/views/accessibility_statement.html [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+  <p class="govuk-body">
+    We’ll continue to investigate the cause of these issues and, if possible, fix them by March 2021.
+  </p>
+
+  <h3 class="heading-small">
+    Status page
+  </h3>
+
+  <p class="govuk-body">
+    The following content on the Notify status page is not compliant with the Web Content Accessibility Guidelines version 2.1 AA standard:
```
Comment (quis) [inline app/templates/views/accessibility_statement.html:144]: I’d suggest adding a link here so people can confirm which page we’re talking about, like we do when talking about the documentation domain at the top of the page:
```suggestion
    The following content on the <a class="govuk-link" href="https://status.notifications.service.gov.uk/">Notify status page</a> is not compliant with the Web Content Accessibility Guidelines version 2.1 AA standard:
```

File: app/templates/views/accessibility_statement.html [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+  </p>
+    
+  <ul class="govuk-list govuk-list--bullet">
+    <li>an incorrect heading hierarchy which fails WCAG 2.1A 1.3.1 success criterion (Info and Relationships)</li>
+    <li>missing level 1 headings (users may not be able to accurately determine the structure of content on the page)</li>
+    <li>our status incident page is missing the main heading and the status home page has an incorrect heading hierarchy – this fails WCAG 2.1A 1.3.1 success criterion (Info and Relationships)</li>
+    <li>our status page ‘subscribe to updates’ link does not contain programmatically discernible link text – tThis fails WCAG 2.1A 1.3.1 success criterion (Info and Relationships)</li>
```
Comment (quis) [inline app/templates/views/accessibility_statement.html:151]:
```suggestion
    <li>our status page ‘subscribe to updates’ link does not contain programmatically discernible link text – This fails WCAG 2.1A 1.3.1 success criterion (Info and Relationships)</li>
```

**Review by quis — APPROVED** [review]

File: app/templates/views/accessibility_statement.html [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+  </p>
+
+  <p class="govuk-body">
+    Screen reader users may find the process of moving templates and folders confusing. This fails <a class="govuk-link" href="https://www.w3.org/TR/WCAG21/#name-role-value">WCAG 4.1.2 success criteria (Name, role, value)</a>. We plan to fix this by November 2020.
+  </p>
+
+  <p class="govuk-body">
+    Screen reader users may find it confusing that a specific part of some pages is focused when they load. This fails <a class="govuk-link" href="https://www.w3.org/TR/WCAG21/#focus-order">WCAG 2.4.3 success criteria (Focus Order)</a>. We plan to fix this by October 2020.
+  </p>
+
```
Comment (karlchillmaid) [inline app/templates/views/accessibility_statement.html:111]:
```suggestion

<p class="govuk-body">
We aim to fix these issues by March 2021.
</p>
```

Comment (karlchillmaid) [reply app/templates/views/accessibility_statement.html:111]: @tombye the PDF, Email and Status page sections all have sentences that commit us to fixing things by a certain point. We should include one here.

Comment (tombye) [reply app/templates/views/accessibility_statement.html:111]: Should we remove the statements at the end of those lines in place of this? I mean lines 101, 105 and 109.

Comment (karlchillmaid) [reply app/templates/views/accessibility_statement.html:111]: No. I think we need each one. The timeframes might be similar, but the details are different.

Comment (karlchillmaid) [reply app/templates/views/accessibility_statement.html:111]: My mistake. I missed those at the end of each line.

We don't need to commit this additional line.

**Pull request merged**